.github/workflows/ci-nix.yml

```diff
@@ -92,6 +92,26 @@ jobs:
         if: (github.ref == 'refs/heads/master' || matrix.build-in-pr) && matrix.run-tests
         run: nix build -L .#ci.cli-test.all
 
+  wasm-tests:
+    name: "Wasm tests"
+    runs-on: buildjet-8vcpu-ubuntu-2004
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: cachix/install-nix-action@v21
+        with:
+          nix_path: nixpkgs=channel:nixos-22.05
+      - uses: cachix/cachix-action@v12
+        with:
+          name: fedimint
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+        continue-on-error: true
+
+      - name: Tests
+        run: nix develop .#crossWasm -c ./scripts/wasm-tests.sh
```

Comment:
I need to verify, but since this is using `workspaceBuild`, we might want to run it in the normal job, after existing tests, so that we are not doing whole workspace build twice.